packages/react/src/ui/Connect/components/Connectors/Connectors.tsx

```diff
@@ -14,34 +59,52 @@ export function Connectors() {
     dialog: { connect },
   } = useConnectUI();
 
+  const [renderConnectorItem] = useState(() =>
+    renderConnector.bind(null, connect, theme),
+  );
+
+  const { native, external } = useMemo(
+    () =>
+      connectors.reduce(
+        (acc, curr) => {
+          if (
+            curr.name.startsWith('Fuel') ||
+            curr.name === 'Bako Safe' ||
+            curr.name === 'Burner Wallet'
+          ) {
+            acc.native.push(curr);
+          } else {
+            acc.external.push(curr);
+          }
```

Comment:
if you think it's a good idea to create a prop in connectors to inform if it's external? it could be a sim `isExternal` prop, then we consider every connector as native but the `isExternal` one

Reply:
Yeah we should do something like this, using the name here is not a good idea

Reply:
done